### DISCUSSION TITLE
Ability to use variables in .npmrc

### DISCUSSION THREAD

**Discussion by maximebochon** [npm/feedback#593] [CLI] [answered]
I would like to be able to use variables from inside the `.npmrc` configuration file.

For instance:
```
proxy=http://user:password@proxy.intra:8000/
http_proxy=${proxy}
https_proxy=${proxy}
```

**Comment by MylesBorins** [comment, marked as answer]
@maximebochon if you would like to propose new functionality to the CLI you should open an issue at https://github.com/npm/rfcs
